hive/src/test/java/org/apache/iceberg/hive/CustomHiveCatalogTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
```

Comment:
nit: Test files name Test as prefix not suffix

Reply:
Fixed